### DISCUSSION TITLE
datePicker, timeInput not working

### DISCUSSION THREAD

**Discussion by laurentwww** [primefaces/community#158] [PrimeFaces] [answered]
I'm using datePicker with Primefaces 10.0.18.
timeInput="true" doesn't work. Is this option available in v. 10 ?

**Comment by melloware** [comment]
according to the 10.0.0 docs its available? https://primefaces.github.io/primefaces/10_0_0/#/components/datepicker?id=datepicker

**Comment by laurentwww** [comment]
Yes, I checked, it's available on 10 version. But it doesn't work.

**Reply by melloware** [reply]
Have you tried it in a newer version?

**Comment by laurentwww** [comment]
No, it's the latest 10 version. I plan to migrate to 11 or more, but need do a lot of change in my code I guess for that.

**Reply by jepsar** [reply, marked as answer]
Don't guess, follow the migration guide. And the latest version is 13.0.1.
